src/styles/calendar.scss

```diff
@@ -0,0 +1,36 @@
+.calendar {
+  position: relative;
+  width: 800px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
```

Comment:
Ensure that the variable `$gap` is defined in the `variables.scss` or an equivalent file. If it's not defined, it will cause a compilation error.